x-pack/plugins/graph/common/check_license.ts

```diff
@@ -6,7 +6,12 @@
 
 import { i18n } from '@kbn/i18n';
 import { ILicense } from '../../licensing/common/types';
-import { assertNever } from '../../../../src/core/utils';
+
+// Can be used in switch statements to ensure we perform exhaustive checks, see
+// https://www.typescriptlang.org/docs/handbook/advanced-types.html#exhaustiveness-checking
+export function assertNever(x: never): never {
+  throw new Error(`Unexpected object: ${x}`);
+}
```

Comment:
Why not import this `assertNever` just like all the otherrs? Is it because it's in `common`?

Reply:
Yep, there isn't currently a common import for src/core and it was just so simple.